### DISCUSSION TITLE
f-droid not up to date

### DISCUSSION THREAD

**Discussion by paulle** [Abrynos/ShoppingList#52] [Q&A] [answered]
The new version (1.12.1) isn't on f-droid.
V 1.11 is on f-droid.

**Comment by Abrynos** [comment, marked as answer]
I've noticed that the F-Droid build is failing because they aren't updating their database to allow current gradle versions very regularly and I'm here using the most recent stable release. I have already opened an issue in their issue tracker, but I have yet to receive an answer. 😕

**Reply by Abrynos** [reply]
Update: The build finally succeeded for version `1.12.1` (as you can see [here](https://monitor.f-droid.org/builds/log/pl.edu.pjwstk.s999844.shoppinglist/14)). Going by how long it took for the last gradle update to be deployed on their server (28 days), I fear future versions might also suffer from some delays over there. Sadly there is nothing I can do to speed this up.

**Comment by paulle** [comment]
Ok, thank you for information

Gesendet von ProtonMail mobile

-------- Original-Nachricht --------

**Comment by paulle** [comment]
There is nothing you do.

Thank you for your efforts.

I'll wait till the new version is on f-droid.

Gesendet von ProtonMail mobile

-------- Original-Nachricht --------
